Resources/Prototypes/Entities/Buildings/power.yml

```diff
@@ -119,7 +119,7 @@
 - type: entity
   parent: WPPnobattery
   id: WPP
-  name: WPP
+  name: wpp
```

Comment:
Think this is an acronym. Same with `APC` and `SMES` below.

Reply:
It's Wireless Power Provider.